ui/main/src/app/business/view/realtimeusers/realtimeusers.view.ts

```diff
@@ -0,0 +1,170 @@
+    private pageLoaded = new ReplaySubject<RealtimePage>(1);
+    private updateInterval;
+
+    constructor(private configServer: ConfigServer,
+                private logger: OpfabLoggerService,
+                private userService: UserService,
+                private entitiesService: EntitiesService,
+    ) {
+        this.init();
+    }
+
+    private init() {
+        this.configServer.getRealTimeScreenConfiguration().subscribe((result) => {
+            if (result.status === ServerResponseStatus.OK && result.data.realTimeScreens.length > 0) {
+                this.realtimePage = new RealtimePage();
+
+                const config = result.data.realTimeScreens;
+                config.forEach((configScreen, index) => {
+
+                    // options
+                    const screenOption = new RealtimePageScreenOption();
+                    screenOption.value = String(index);
+                    screenOption.label = configScreen.screenName;
+                    this.realtimePage.screenOptions.push(screenOption);
+
+                    // screens
+                    const screen = new RealtimePageScreen();
+                    screen.name = configScreen.screenName;
+                    screen.onlyDisplayUsersInGroups = configScreen.onlyDisplayUsersInGroups ?? [];
+                    configScreen.screenColumns.forEach((configColumn) => {
+
+                        // columns
+                        const screenColumn = new RealtimePageScreenColumn();
+                        configColumn.entitiesGroups.forEach((configEntityGroup) => {
+
+                            // entitiesGroups
+                            const entityGroup = new RealtimePageEntityGroup();
+                            entityGroup.name = configEntityGroup.name;
+                            configEntityGroup.entities.forEach((configEntity) => {
+
+                                // lines
+                                const line = new RealtimePageLine();
+                                line.entityId = configEntity;
+                                line.entityName = this.entitiesService.getEntityName(configEntity);
+                                line.connectedUsersCount = 0;
+                                line.connectedUsers = '';
+                                entityGroup.lines.push(line);
+                            });
+                            screenColumn.entitiesGroups.push(entityGroup);
+                        });
+                        screen.columns.push(screenColumn);
+                    });
+                    this.realtimeScreens.push(screen);
+                });
+                this.logger.info('realTimeScreens config loaded');
+                this.updateConnectedUsers();
+                this.startUpdateInterval(2000);
+            } else {
+                this.logger.error('realTimeScreens config could not be loaded');
+            }
+            this.pageLoaded.next(this.realtimePage);
```

Comment:
add : "this.pageLoaded.complete(); "

As you send only one event , it is safer to complete to free memory (and avoid memory leak)